docs/configure-and-manage/manage/qlight-node.md

```diff
@@ -14,41 +14,42 @@ The server node is usually set up to support [multiple private states](../../con
 
 Configure the qlight client using the following command line options:
```

Comment:
Since the options descriptions are simply repeated here, I recommend changing to:

```suggestion
Configure the qlight client using the [`--qlight.client.*`](../../reference/cli-syntax.md#qlightclient) command line options.
```

And deleting the bulleted list.